src/utils/validations.ts

```diff
@@ -111,11 +111,11 @@ export function validateVaultWithdraw(props: ValidateVaultWithdrawProps): Valida
   if (yvTokenAmount.isZero()) return {};
 
   if (amountInWei.lt(0)) {
-    return { error: 'INVALID AMOUNT' };
+    return { error: 'Invalid amount' };
```

Comment:
I feel these errors could be a bit more meaningful if we were to use the logic to get here. Example:
```suggestion
    return { error: `Invalid amount: ${amountInWei}` };
```

Reply:
in this case amountInWei would be just what the user set on the input, I don't know if it makes much sense to tell the user that

Reply:
Good point 😅 I forgot these were coming from there. All good, I had a look at the others and the user should be able to deduce why the error is happening. Resolving!